src/credentials/CHIPCert.h

```diff
@@ -544,6 +557,14 @@ CHIP_ERROR ValidateChipRCAC(const ByteSpan & rcac);
  */
 CHIP_ERROR ValidateChipNetworkIdentity(const ByteSpan & cert);
 
+/**
+ * Convenience variant of `ValidateChipNetworkIdentity` that, upon successful validation, also
+ * calculates the key identifier for the Network (Client) Identity.
+ * @see ValidateChipNetworkIdentity
+ * @see ExtractIdentifierFromChipNetworkIdentity
+ */
+CHIP_ERROR ValidateChipNetworkIdentity(const ByteSpan & cert, MutableCertificateKeyId keyId);
```

Comment:
I'm not a huge fan of using "Chip" in the new APIs here, but I guess I can live with it....

Reply:
"Chip" here in the sense that it's in the TLV format, to align with all the other similar functions.